### ISSUE TITLE
missing subfield.dlabel.gii outputs

### ISSUE THREAD

**Issue by jordandekraker** [khanlab/hippunfold#217]
This rule is not being run in the singularity container v1.2.0, and I think even in the current main branch it is not run. 

https://github.com/khanlab/hippunfold/blob/13d26caa4a35a9361bf64eae35ab3b5ebdefbdbc/hippunfold/workflow/rules/gifti.smk#L500

TODO: add this to a get_final rule in common.smk

**Comment by akhanf** [comment]
Are you sure you're using the v1.2.0? This bug was fixed specifically in #211, and when I ran v1.2.0 a couple days ago it worked for me, generated:

e.g.:
```
hippunfold/sub-ADNI011S4547/ses-M66/surf/sub-ADNI011S4547_ses-M66_hemi-L_space-T1w_den-0p5mm_label-hipp_atlas-bigbrain_subfields.label.gii
hippunfold/sub-ADNI011S4547/ses-M66/surf/sub-ADNI011S4547_ses-M66_hemi-R_space-T1w_den-0p5mm_label-hipp_atlas-bigbrain_subfields.label.gii
hippunfold/sub-ADNI011S4547/ses-M66/surf/sub-ADNI011S4547_ses-M66_space-T1w_den-0p5mm_label-hipp_atlas-bigbrain_subfields.dlabel.nii
```

Also, do you mean `dlabel.nii`? or `label.gii`?  I am guessing the `dlabel.gii` in your issue title is a typo?

**Comment by jordandekraker** [comment]
I did mean `label.gii`, thanks. And yeah I just checked, my singularity image is actually from just before that PR was merged.
